mpags-cipher.cpp

```diff
@@ -0,0 +1,106 @@
+#include <iostream>
+#include <string>
+#include <cctype>
+#include <vector>
+int main(int argc, char* argv[])
+{
+    const std::vector<std::string> cmdLineArgs { argv, argv+argc }; 
+
+    long unsigned int length { cmdLineArgs.size() };
```

Comment:
Good storing the length in a variable, since it is used in a few places.  It can and should be made `const` because you don't want to change it.  There is also a convenient typedef for `long unsigned int`, which is `std::size_t`, which is a bit less typing.
```suggestion
    const std::size_t length { cmdLineArgs.size() };
```